### PR TITLE
Added aws sqs keda connector documentation 

### PR DESCRIPTION
**Pull request by girishg4t**
Added documentation for aws sqs connector.
it basically tells how we can connect and test aws sqs connector

### CODE REVIEW

File: docs/content/en/docs/triggers/message-queue-trigger/aws-sqs.md [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+```
+
+### Connecting via trigger
+
+We have both the functions ready but the connection between them is the missing glue.
+Let's create a message queue trigger which will invoke the consumerfunc every time there is a message in `input` queue.
+The response will be sent to `output` queue and in case of consumerfunc invocation fails, the error is written to `error` queue.
+
+```bash
+$ fission mqt create  --name sqstest --function consumerfunc --mqtype aws-sqs-queue --topic input --resptopic output --mqtkind keda --errortopic error --maxretries 3 --metadata queueURL=https://sqs.ap-south-1.amazonaws.com/xxxxxxxx/input --metadata awsRegion=ap-south-1 --cooldownperiod=30 --pollinginterval=5 --secret awsSecrets
```
Comment (vishal-biyani) [inline docs/content/en/docs/triggers/message-queue-trigger/aws-sqs.md:138]: Can we add a comment here pointing to https://keda.sh/docs/2.0/scalers/aws-sqs/ for all metadata fields used?

**Review by vishal-biyani — CHANGES_REQUESTED** [review]
Some minor comments, but look good otherwise

File: docs/content/en/docs/triggers/message-queue-trigger/aws-sqs.md [hunk 2]
```diff
@@ -0,0 +1,203 @@
+---
+title: "AWS SQS"
+draft: false
+weight: 78
+---
+
+This tutorial will demonstrate how to use a AWS SQS trigger to invoke a function.
+We'll assume you have Fission and Kubernetes installed with AWS SQS Queue integration installed.
```
[the remark this answers is not in the document]
Comment (girishg4t) [reply docs/content/en/docs/triggers/message-queue-trigger/aws-sqs.md]: Yes, removed that not needed

File: docs/content/en/docs/triggers/message-queue-trigger/aws-sqs.md [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+If not, please head over to the [install guide]({{% ref "../../installation/_index.en.md" %}}).
+
+You will also need AWS SQS setup which is reachable from the Fission Kubernetes cluster.
+
+## Installation
+
+If you want to setup SQS on the Kubernetes cluster, you can use the [information here](https://github.com/localstack/localstack) or you can create queue using your aws account [docs](https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/sqs-setting-up.html).
```
[the remark this answers is not in the document]
Comment (girishg4t) [reply docs/content/en/docs/triggers/message-queue-trigger/aws-sqs.md]: Yes, added note

**Comment by vishal-biyani** [comment]
Hey @girishg4t This should go in the section under Keda Triggers and not in Triggers section: 
image

File: docs/content/en/docs/triggers/message-queue-trigger/aws-sqs.md (hunk 1 above)
Comment (girishg4t) [reply docs/content/en/docs/triggers/message-queue-trigger/aws-sqs.md]: added description for the field required and removed unwanted

**Comment by vishal-biyani** [comment]
Thanks a lot @girishg4t! 🎉

**Pull request merged**